packages/core-typings/src/IUser.ts

```diff
@@ -191,7 +192,7 @@ export interface IRegisterUser extends IUser {
 }
 
 export const isRegisterUser = (user: IUser): user is IRegisterUser => user.username !== undefined && user.name !== undefined;
-export const isUserFederated = (user: Partial<IUser>) => 'federated' in user && user.federated === true;
+export const isUserFederated = (user: Partial<IUser> | Partial<Serialized<IUser>>) => 'federated' in user && user.federated === true;
```

Comment:
`isUserFederated` is not actually an assertion function, this function is actually simple useless, and  it should not be used since the `federated` prop is defined on the `IUser` type, so just do `Boolean(user.federated)` instead